spaces/domains.json

```
@@ -4,6 +4,7 @@
   "congress.subx.finance": "subx.eth",
   "dao.kawatoken.io": "kawadao.eth",
   "vote.blockadvocates.org": "blockadvocates.eth",
+  "polls.chain.com":"chaingov.eth",
```

Please add a space between sub-domain and ENS.
```suggestion
  "polls.chain.com": "chaingov.eth",
```